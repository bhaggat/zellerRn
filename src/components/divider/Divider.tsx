import {StyleSheet, View, ViewProps} from 'react-native';

export default function Divider({style, ...props}: ViewProps) {
  return <View style={[styles.divider, style]} {...props} />;
}

const styles = StyleSheet.create({
  divider: {
    backgroundColor: '#E2E8F0',
    width: '100%',
    height: 1,
    marginVertical: 10,
  },
});
