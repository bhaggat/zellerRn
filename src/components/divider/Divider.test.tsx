import {render} from '@testing-library/react-native';
import Divider from './Divider';

describe('Divider Component', () => {
  describe('Snapshot', () => {
    test('matches', () => {
      const tree = render(<Divider />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
