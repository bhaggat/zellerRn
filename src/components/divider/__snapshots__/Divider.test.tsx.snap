// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Component Render Component render 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#E2E8F0",
        "height": 1,
        "marginVertical": 10,
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;

exports[`Divider Component Snapshot matches 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#E2E8F0",
        "height": 1,
        "marginVertical": 10,
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
